File: app/src/main/java/com/demo/demoapplication/ui/fragments/slideshow/SlideshowViewModel.java
package com.demo.demoapplication.ui.fragments.slideshow;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SlideshowViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public SlideshowViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Hello world 2");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/demo/demoapplication/ui/fragments/slideshow/SlideshowViewModel.kt
package com.demo.demoapplication.ui.fragments.slideshow

import androidx.lifecycle.ViewModel
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.LiveData

class SlideshowViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "Hello world 2"
    }
}